### PR TITLE
docs: cleanup

### PR DESCRIPTION
**Pull request by jorgemmsilva**
also includes some outdated config fixes

### CODE REVIEW

File: documentation/docs/guide/chains_and_nodes/chain-management.md [hunk 1]
```diff
@@ -65,19 +57,19 @@ This node won't be "officially" recognized by the committee, but will still be a
...
-- If the chain governor address is a regular user wallet, you can issue the rotation transaction using wasp-cli:
+- If the chain governor address is a regular user wallet (that you control), you can issue the rotation transaction using wasp-cli:
 
-```bash
+```shell
 wasp-cli chain rotate <new controller address>
 ```
 
-or
+or, if you controll it and wish to give the control of the alias output to the chain committe:
```
Comment (dessaya) [inline documentation/docs/guide/chains_and_nodes/chain-management.md:71]:
```suggestion
or, if you control it and wish to give the control of the alias output to the chain committe:
```

File: documentation/docs/guide/chains_and_nodes/setting-up-a-chain.md [hunk 2]
```diff
@@ -48,78 +40,75 @@ Example response:
...
+```
 
 ## Testing If It Works
 
 You can check that the chain was properly deployed in the Wasp node dashboard
 (e.g. `127.0.0.1:7000`). Note that the chain was deployed with
 some [core contracts](../core_concepts/core_contracts/overview.md).
 
-## Video Tutorial
+you should have an EVM-JSONRPC server opened on:
```
Comment (dessaya) [inline documentation/docs/guide/chains_and_nodes/setting-up-a-chain.md:100]:
```suggestion
You should also have an EVM-JSONRPC server opened on:
```

File: documentation/docs/guide/chains_and_nodes/setting-up-a-chain.md [hunk 3]
```diff
@@ -48,78 +40,75 @@ Example response:
...
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/3mLpV_neB6I" title="Setting up Wasp Chain" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+```info
+<wasp API URL>/chain/<CHAINID>/evm/jsonrpc
+```
 
 ### Deploying a Wasm Contract
 
+:::warning
+The WASM VM is experimental, however, similar commands can be used to interact with the core contracts
```
Comment (dessaya) [inline documentation/docs/guide/chains_and_nodes/setting-up-a-chain.md:109]:
```suggestion
The WASM VM is experimental. However, similar commands can be used to interact with the core contracts.
```

The second part of the warning is unclear: if the section is about deploying wasm contracts why is there mention of "similar commands to interact with core contracts"?

Also, is the experimental warning really necessary? I mean, the wasp version tag already includes `alpha` :stuck_out_tongue:

File: documentation/docs/guide/chains_and_nodes/wasp-cli.md [hunk 4]
```diff
@@ -14,10 +14,10 @@ keywords:
 
 Step-by-step instructions on how to use wasp-cli to interact with Wasp nodes on the Hornet network.
 
-## Requirements
+## Download wasp-cli
 
-After going through the instructions on [Running a node](./running-a-node.md), you should have the `wasp-cli` binary
-available in your system.
+Download the latest wasp-cli binary from the repo [relases page](https://github.com/iotaledger/wasp/releases).
+(for ease of use, its recommend adding wasp-cli to your system PATH)
```
Comment (dessaya) [inline documentation/docs/guide/chains_and_nodes/wasp-cli.md:20]:
```suggestion
(For ease of use, it is recommend to add `wasp-cli` to your system `PATH`).
```

File: documentation/docs/guide/wasm_vm/intro.mdx [hunk 5]
```diff
@@ -1,21 +1,27 @@
...
 
 description: IOTA Smart Contracts (ISC) provides a very flexible way of programming smart contracts by providing an API to a sandboxed environment that allows you to interact deterministically and without any security risks with ISC-provided functionality.
 
 image: /img/wasm_vm/IscHost.png
 ---
 
+:::warning
+The Wasm VM is still in an experimental state, it is a testament to the "VM plugin" architecture of ISC.
+
+Experiment away, but please don't rely on it for any valuable applications (stick to EVM for now)
```
Comment (dessaya) [inline documentation/docs/guide/wasm_vm/intro.mdx:22]:
```suggestion
Experiment away, but please don't rely on it for any valuable applications (stick to EVM for now).
```

Same as above, I'm not sure EVM can be consedered non-"experimental"...

File: tools/monitoring/grafana/dashboards/general_dashboard.json [hunk 6]
```diff
@@ -19,7 +19,7 @@
     ]
   },
   "editable": true,
-  "gnetId": null,
+  "gPeeringURL": null,
```
Comment (dessaya) [inline tools/monitoring/grafana/dashboards/general_dashboard.json:22]: typo here?

**Review by dessaya — APPROVED** [review]

File: documentation/docs/guide/wasm_vm/intro.mdx (hunk 5 above)
Comment (jorgemmsilva) [reply documentation/docs/guide/wasm_vm/intro.mdx:22]: yeah, but we want people to build on EVM, not WASM for the time being :disappointed:

**Pull request merged**